dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/UnpackagedTests/MrtCoreUnpackagedTests.csproj

```diff
@@ -50,8 +50,14 @@
     <Reference Include="Wex.Logger.Interop, Version=10.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\..\packages\Microsoft.Taef.10.58.210222006-develop\lib\net45\Wex.Logger.Interop.dll</HintPath>
     </Reference>
+    <Reference Include="Windows.Foundation.FoundationContract">
+      <HintPath>C:\Program Files (x86)\Windows Kits\10\References\10.0.19041.0\Windows.Foundation.FoundationContract\4.0.0.0\Windows.Foundation.FoundationContract.winmd</HintPath>
```

Comment:
Not sure, but does this need to have a hard-coded path to the SDK? (What if the SDK is installed on another drive?)
Or is this just a "hint"?

Reply:
I didn't find a definition of metadata location.

Reply:
Yeah, I don't see any easy way to get this either.

Reply:
Thanks Jeff for checking this!